react/components/ReviewBlock.tsx

```diff
@@ -409,9 +410,14 @@ const ReviewBlock: FunctionComponent<WrappedComponentProps & any> = ({
     }
   }
 
-  const tableSchema = {
+  let tableSchema = {
```

Comment:
Hi @whc105 I cannot create a default tableSchema because none of the properties are optional (following the way it was done previously)(image attached)

I see three possible ways to do it, based on a research I did.
1. Create an interface with all properties as optional
2. Change the option "strictNullChecks" to false in the tsconfig.json
3. Keep the way it is in this commit, adding the properties if they are not setted to be hidden.

Which one should I follow?
Doing it the way it is now affects the performance in any way?

![image](https://user-images.githubusercontent.com/34545119/154761702-4e2066bd-f558-426e-8813-550c4346c08d.png)

Reply:
Hey @JooLuiz 
We can go with what you have right now. I realized deleting might not be as viable compared to just adding. However, it would be nice to change the hiddenColumns to a Set.